controller-manager/src/main/resources/templates/PrometheusRules-enmasse.yaml

```diff
@@ -20,6 +20,9 @@ spec:
     - record: enmasse_component_health
       expr: up{job="address-space-controller"} or on(namespace) (1- absent(up{job="address-space-controller"}) )
 
+    - record: enmasse_component_health
+        expr: up{job="enmasse-operator"} or on(namespace) (1- absent(up{job="enmasse-operator"}) )
```

Comment:
The operator is actually exporting metrics with the label `job="enmasse-operator-metrics"`.